uwblueprint/rowan-house
# Implement course previews on admin homepage with dummy data

## Implementation Description
Implement course preview block with module components

## Screenshots

https://user-images.githubusercontent.com/40974372/158055880-767e3a21-4a3e-453d-8199-5e92454fdcef.mov


## What should reviewers focus on?
- Proper usage of Chakra components, units, general front-end practice

## Testing Procedure
- What test(s) should we run to make sure your code works?
   - What ways could the input be wrong?
   - Is it secure? Could nefarious users access/break it?

No tests at the moment

## Things to Note
- Additional dependencies/libraries you've added? Nope
- Things you'd want to know when coming back to the code after a few months
  - Anything too hardcoded?

## Checklist
